test/functional/README.md

```diff
@@ -51,10 +51,13 @@ don't have test cases for.
 
 #### General test-writing advice
 
+- Instead of inline comments or no test documentation at all, prefer to log the comments to the test log. E.g.
+  `self.log.info('Creating enough transactions to fill a block'`. Logs make the test code easier to read and the test
```

Comment:
According to [review feedback](https://github.com/bitcoin/bitcoin/pull/17327#pullrequestreview-310002880), it may be better (and shorter) to use the indicative, e.g. "Create..." rather than "Creating..."

Reply:
(That comment proposes some good guidelines for your consideration:)
- "test x" rather than "testing x"
- logs can replace docstrings/comment blocks
- no need for trailing '...' ellipses

Reply:
Thanks, made shorter